OneSignalSDK/unittest/src/test/java/com/onesignal/OneSignalPackagePrivateHelper.java

```diff
@@ -513,8 +513,12 @@ public OSTestInAppMessageAction(JSONObject json) throws JSONException {
 
    public static void dismissCurrentMessage() {
       com.onesignal.OSInAppMessage message = OneSignal.getInAppMessageController().getCurrentDisplayedInAppMessage();
-      if (message != null)
+      if (message != null) {
          OneSignal.getInAppMessageController().messageWasDismissed(message);
+      }
```

Comment:
since re-formatting code will put the else after the brace, can we keep the same code styling? 

  ```
if (message != null) {         
       OneSignal.getInAppMessageController().messageWasDismissed(message);    
  } else {
     ....
  }
```

Reply:
Fixed